src/Entity/AddressValidation/AVAddress.php

```diff
@@ -0,0 +1,73 @@
+<?php namespace Ups\Entity\AddressValidation;
+
+class AVAddress
+{
+    public $addressClassification;
```

Comment:
Can you add docs to these parameters? It makes it easier for IDE's to understand what is potentially coming in it.